root/artist/header.tt

```diff
@@ -1,10 +1,10 @@
-<div class="artistheader">
-    <h1>[% link_entity(artist) %]</h1>
+[% header = artist.type_name _ 'icon' %]
+<div class = "artistheader [% header.lower() %]">
```

Comment:
This also doesn't handle artists that have no type. You need to add `IF artist.type_name` at the end of the expression.